oracle/README.md

```diff
@@ -321,6 +337,79 @@ Verifies the database is available and accepting connections.
 
 ## Troubleshooting
 
+### Common Problems
+#### Oracle Instant Client
+- Verify that both the Oracle Instant Client and SDK files are located in the same directory.
+The structure of the directory should look similar:
+```text
+|____libociei.dylib
```

Comment:
I believe there should be two directories in this folder. Can we show themp like so:

```
|____dir1/
|____dir2/
|____file1
|____libocci.dylib.10.1
...
```